### PR TITLE
Chore: Convert VerticalBar component to typescript

### PR DESCRIPTION
**Pull request by dougfabris**

### CODE REVIEW

**Review by gabriellsh — CHANGES_REQUESTED** [review]

File: client/components/VerticalBar/VerticalBarActions.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { ButtonGroup } from '@rocket.chat/fuselage';
+import React, { FC, memo } from 'react';
+
+const VerticalBarActions: FC<{ children: React.ReactNode }> = (props) => (
```
Comment (gabriellsh) [inline client/components/VerticalBar/VerticalBarActions.tsx:4]: Do not use "React.*", always import what you're referencing. React is to drop support to this pattern.

File: client/components/VerticalBar/VerticalBarHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Box, Margins } from '@rocket.chat/fuselage';
+import React, { FC, memo } from 'react';
+
+const VerticalBarHeader: FC<{ children: React.ReactNode }> = ({ children, ...props }) => (
```
Comment (gabriellsh) [inline client/components/VerticalBar/VerticalBarHeader.tsx:4]: Same as the other one.

**Review by gabriellsh — CHANGES_REQUESTED** [review]

File: client/components/VerticalBar/VerticalBarInnerContent.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Box } from '@rocket.chat/fuselage';
+import React, { FC, memo } from 'react';
+
+const VerticalBarInnerContent: FC = (props) => (
```
[the remark this answers is not in the document]
Comment (dougfabris) [reply client/components/VerticalBar/VerticalBarInnerContent.tsx:4]: Thanks buddy!

File: client/components/VerticalBar/VerticalBarActions.tsx (hunk 1 above)
Comment (dougfabris) [reply client/components/VerticalBar/VerticalBarActions.tsx:4]: Great observation! Was a lack of attention here!

**Review by tassoevan — APPROVED** [review]

**Review by gabriellsh — APPROVED** [review]

**Pull request merged**